src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -85,9 +108,19 @@ function BaseHeader(props: HeaderProps) {
       <div className={classes.toolbar}>
         <div className={classes.left}>{left}</div>
         <div className={classes.right}>{right}</div>
+        <Menu anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={closeMenu}>
```

Comment:
It'd be more legible to hide this JSX behind a variable name like `accountMenu` or something like that. This can get out of hand pretty quickly if stuff keeps getting appended.

Reply:
Good call! If I understood you correctly, I _think_ I fixed this now.